cterasdk/edge/services.py

```diff
@@ -122,6 +122,10 @@ def _connect_to_services(self, param, ctera_license):
         task = self._attach(param)
         try:
             TaskManager.wait(self._gateway, task)
+            track(self._gateway, '/status/services/CTERAPortal/connectionState', [enum.ServicesConnectionState.Connected], [],
```

Comment:
Sorry for nitpicking, but aren't these statuses "progress"? (Not that it matters to much to the outcome)

Reply:
True. The `edge/sync` module is the one that may rely on transient states. I changed the order